content/docs/intro/concepts/state.md

```diff
@@ -99,6 +99,17 @@ At a glance, the Pulumi Service backend provides the following benefits:
 - Full deployment history for auditing and rollback purposes
 - Concurrent state locking to prevent corrupting your infrastructure state in a team environment
 - Advanced policy and RBAC (Role Based Access Control)
+- Stack access control, including information on who owns a stack, who last modified stack, and what time the stack was modified
+- Refresh config from the last deployment
+- Secrets handled by a key managed by the service or the option to use passpharse-based key
+
+### Self-managed backend risks
```

Comment:
I am not a fan of using the word "risks" here - they are not risks they just requirement that we need to ensure people understand the differences